core/src/com/unciv/ui/tilegroups/layers/TileLayerUnitFlag.kt

```diff
@@ -71,32 +80,28 @@ class TileLayerUnitFlag(tileGroup: TileGroup, size: Float) : TileLayer(tileGroup
     }
 
     private fun getAirUnitTable(unit: MapUnit): Table {
-        val holder = Table()
 
-        val iconColor = unit.civInfo.nation.getInnerColor()
-        val bgColor = unit.civInfo.nation.getOuterColor()
+        val iconColor = unit.civInfo.nation.getOuterColor()  //Color.WHITE.cpy().apply { a = 0.9f }
+        val bgColor = unit.civInfo.nation.getInnerColor() //Color.BLACK.cpy().apply { a = 0.9f }
```

Comment:
Not something that should be put in this PR but it seems to me that there should be a standard class for icons like this on the world screen. Something like `WorldScreenCivilizationIcon(civ: Civilization, shape: [some shape enum like square or circle or diamond or rounded horizontal rectangle {pill shape?} depending on need])` that looks for the `civ`'s main and secondary colors and can be applied to all sorts of image groups.

Reply:
World scree Units icons are visualized already by `UnitGroup` class, which selects shape and color based on type/status/civ.